src/components/expanded-state/custom-gas/FeesPanel.js

```diff
@@ -505,9 +523,7 @@ export default function FeesPanel({
           />
         </PanelColumn>
       </MiddlePanelRow>
-      <Animated.View style={maxPriorityWarningsStyle}>
-        {renderWarning(maxPriorityFeeError, maxPriorityFeeWarning)}
-      </Animated.View>
+      <Row>{renderWarning(maxPriorityFeeError, maxPriorityFeeWarning)}</Row>
```

Comment:
question via slack

Reply:
from what understand we need the animations and this is removing it cc @christianbaroni